# This is an action definition file, which has three parts:
# The goal, the result, and the feedback

# Part 1: The goal
# The amount of time we want to wait
duration time_to_wait
---

# Part 2: The result to be sent by the server upon completion
# How much time we waited & how many updates we provided along the way
duration time_elapsed
uint32 updates_sent
---

# Part 3: The feedback to be sent periodically by the server during execution
# The amount of time that has elapsed from the start and remaining until done
duration time_elapsed
duration time_remaining

